data-validation/root.crt

```diff
@@ -0,0 +1 @@
+../root.crt
```

Comment:
@jcampbell as an alternative to these symlinks, someone w/ admin privs could edit the `COCKROACH_CONNECTION_STRING` to use `../root.crt` as the sslrootcert path.

Reply:
Makes sense. I'm just going to leave as is for now, but happy to adjust later. Is it different because we're now running in the subdirectory? Is that a requirement of the pipenv approach?

Reply:
Yeah, it's because I set a different working-directory. All good!